scripts/install-chef-suse.sh

```diff
@@ -911,7 +911,7 @@ fi
 
 # Use current time zone
 (
-    . /etc/sysconfig/clock || : Ignoring missing /etc/sysconfig/clock
+    TIMEZONE=$(LC_ALL=C timedatectl status | sed -e "/Time zone:/ s/^.*: \([^ ]\+\).*/\1/; t; d")
```

Comment:
It's slightly more complex than that, sadly.

Cloud 6, so SP1:

root@crowbar:~ # LC_ALL=C timedatectl status | sed -e "/Time zone:/ s/^.*: \([^ ]\+\).*/\1/; t; d"
root@crowbar:~ # LC_ALL=C timedatectl status | sed -e "/Time \?zone:/ s/^.*: \([^ ]\+\).*/\1/; t; d"
UTC

Cloud 7, so SP2:

root@crowbar:~ # LC_ALL=C timedatectl status | sed -e "/Time zone:/ s/^.*: \([^ ]\+\).*/\1/; t; d"
Etc/UTC
root@crowbar:~ # LC_ALL=C timedatectl status | sed -e "/Time \?zone:/ s/^.*: \([^ ]\+\).*/\1/; t; d"
Etc/UTC

So not only did the field change between SP1 and SP2, but the output as well.